.travis.yml

```diff
@@ -0,0 +1,26 @@
+dist: trusty
+language: php
+
+php:
+    - 7.2
+    - 7.3
+
```

Comment:
We need to remember to add php 7.4 here in 2 weeks!

Reply:
Sure, I will create a reminder now 😄